gdp-src-build/conf/templates/bblayers.inc

```diff
@@ -27,3 +27,5 @@ BBLAYERS_NON_REMOVABLE ?= " \
   ${OEROOT}/meta-yocto \
   ${OEROOT}/../meta-ivi/meta-ivi \
   "
+
```

Comment:
I would suggest a comment here to explain the usage:

> \# Include local developer settings.  This can be used to locally modify the
> \# setup of mirrors, caching and any other bitbake variables.
> \# If the file does not exists the include statement has no effect.
> include ${TOPDIR}/developer.conf